html/global_attributes.json

```
@@ -846,6 +846,53 @@
           }
         }
       },
+      "enterkeyhint": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": "77"
+            },
+            "chrome_android": {
+              "version_added": "77"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": "13.1"
+            },
+            "safari_ios": {
+              "version_added": "13.1"
```

Safari iOS versions are based upon iOS versions rather than Safari versions.  Can we fix that?

```suggestion
              "version_added": "13.4"
```